lib/location.js

```diff
@@ -97,6 +97,7 @@ var LocationState = State.extend(function (self, name, opts) {
         characters_per_page:160,
         next_text:"Next",
         previous_text:"Previous",
+        skip_text: "Skip",
```

Comment:
display s. Skip if skip_test is null or is not null? Where would one make it not null?